.circleci/config.yml

```diff
@@ -3,7 +3,7 @@ version: 2
 # https://circleci.com/blog/circleci-hacks-reuse-yaml-in-your-circleci-config-with-yaml/
 defaults: &defaults
   docker:
-    - image: cimg/go:1.17.13
+    - image: cimg/go:1.19
```

Comment:
i have to update the docker image here for a go version that is higher, otherwise i am having linter issue, i think the golangci-linter version inside of cimg/go:1.17.13 is older.